server/channels/api4/ldap_test.go

```diff
@@ -309,4 +309,28 @@ func TestAddUserToGroupSyncables(t *testing.T) {
 	resp, err = th.SystemAdminClient.AddUserToGroupSyncables(context.Background(), user.Id)
 	require.NoError(t, err)
 	CheckOKStatus(t, resp)
+
+	t.Run("should sync SAML users when SamlSettings.EnableSyncWithLdap is true", func(t *testing.T) {
```

Comment:
Nit-pick: Nice to see that you introduce a subtest. Maybe we also convert the other to a sub test while we are here?